entrypoint.sh

```diff
@@ -77,21 +77,25 @@ isBranchBlocked() {
     # ie the head of the 'target' branch is not an ancester of the 'after target' branch
     if git merge-base --is-ancestor \
         $(git rev-parse origin/${TARGET_BRANCH}) \
-        $(git rev-parse origin/${AFTER_TARGET_BRANCH})
-    then
+        $(git rev-parse origin/${AFTER_TARGET_BRANCH}); then
```

Comment:
as a general stylistic item, i would prefer to keep the "then" on an alternate line as it makes it easier to see the difference between the condition and the body

Reply:
Appears to be an autoformat thing going on here, let me see if it will let me reverse it

Reply:
@davidjdixon Not sure what formatting rules are doing this, but it should be reverted now.